wgsl/index.bs

```diff
@@ -7913,56 +7913,64 @@ variables=] for each root identifier.
 
 Two root identifiers <dfn noexport>alias</dfn> when they have the same
 [=originating variable=].
-It is a [=dynamic error=] if:
-* more than one aliased root identifiers access the same [=memory locations=], and
-* at least one access is a [=write access|write=], and
-* both accesses occur during the execution of the same call of the function
-
-Note: This aliasing restriction applies to memory locations written by
-[=function calls=] made in the function.
-
-Note: [=Originating variables=] cannot have aliased [=memory locations=].
-See [[#var-decls]] and [[#resource-interface]].
-
-<div class='example wgsl' heading='Aliased memory views'>
+A WGSL program [=shader-creation error|must not=] contain any potential aliases.
+The program is analyzed from the leaves of the callgraph upwards (e.g. topological order).
+For each function the analysis records the following sets:
+* [=module scope|Module-scope=] variables that are [=write access|written=].
+    Including any module-scope variables that are written in functions called from this function.
+* [=module scope|Module-scope=] variables that are [=read access|read=].
+    Including any module-scope variables that are read in functions called from this function.
+* Pointer parameters that are [=write access|written=].
```

Comment:
Editorial:  The parameters themselves are constants.  It's the fact the code writes *through* the pointers.

Is this any better?

> * The pointer parameters that are root identifiers for memory views through which writes occur, in this function or in called functions.
> * The pointer parameters that are root identifiers for memory views through which reads occur, in this function or called functions.

It's stilted but precise.